src/main/java/racing/model/CurrentLocation.java

```diff
@@ -0,0 +1,47 @@
+package racing.model;
+
+import java.util.Objects;
+
+public class CurrentLocation {
+
+    private static final int Min_Location = 0;
+
+    private int currentLocation;
+
+    public CurrentLocation(int startLocation) {
+        validateLocation(startLocation);
+        this.currentLocation = startLocation;
+    }
+
+    public void geStraight(){
+        this.currentLocation ++;
+    }
+    public int compareCurrentLocation (CurrentLocation o){
+        return Integer.compare(this.currentLocation, o.currentLocation);
+    }
+    public int getCurrentLocation(){
+        return this.currentLocation;
+    }
+    public boolean isSameCurrentLocation(CurrentLocation comparisonCurrentLocation){
+        return this.currentLocation == comparisonCurrentLocation.currentLocation;
+    }
+    private void validateLocation(int startLocation) {
+        if (startLocation < 0) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+
```

Comment:
```suggestion
    public void geStraight(){
        this.currentLocation ++;
    }
    
    public int compareCurrentLocation (CurrentLocation o){
        return Integer.compare(this.currentLocation, o.currentLocation);
    }
    
    public int getCurrentLocation(){
        return this.currentLocation;
    }
    
    public boolean isSameCurrentLocation(CurrentLocation comparisonCurrentLocation){
        return this.currentLocation == comparisonCurrentLocation.currentLocation;
    }
    
    private void validateLocation(int startLocation) {
        if (startLocation < 0) {
            throw new IllegalArgumentException();
        }
    }

```

auto formatting 기능 단축키를 찾아보고 적극 활용해 보아요. 

`geStraight()`는 오타가 있는 것이 아닐까요?